scripts/zones/Western_Altepa_Desert/npcs/_3h5.lua

```diff
@@ -25,9 +28,19 @@ entity.onTrigger = function(player, npc)
         GetNPCByID(ID.npc.ALTEPA_GATE + 7):getAnimation() == xi.anim.OPEN_DOOR and
         GetNPCByID(ID.npc.ALTEPA_GATE + 8):getAnimation() == xi.anim.OPEN_DOOR
     then
-        local openTime = math.random(15, 30) * 60
+        local openTime = math.random(15, 30) * 1000 * 60
+
         for i = ID.npc.ALTEPA_GATE, ID.npc.ALTEPA_GATE + 8 do
-            GetNPCByID(i):openDoor(openTime)
+            local door = GetNPCByID(i)
+
+            -- openDoor used to (read: a decade ish ago before Sol) allow calls to doors already open and set a timer to close them
```

Comment:
side note: want that functionality back :( 

likewise closeDoor used to work on already closed doors and then open them after the time ended.